src/crud/useSharedResUpdate.ts

```diff
@@ -35,16 +39,10 @@ export const useSharedResUpdate = () => {
       const catRec = findRecord(memory, 'artifactcategory', category) as
         | ArtifactCategory
         | undefined;
-      if (catRec) {
-        ops.push(
-          t
-            .replaceAttribute(
-              sharedResource.relationships.passage.data as RecordIdentity,
-              'reference',
-              `NOTE ${catRec.attributes.categoryname}`
-            )
-            .toOperation()
-        );
+      if (catRec && onUpdRef) {
+        const passage = sharedResource.relationships.passage
+          .data as RecordIdentity;
+        onUpdRef(passage.id, `NOTE ${catRec.attributes.categoryname}`);
```

Comment:
since the category name appears to the user, this should be localized

Reply:
yes, I have done this now.